acmw-db-app/pages/forgotPassword.js

```diff
@@ -0,0 +1,99 @@
+import Head from 'next/head'
+import styles from '../styles/Reset.module.css'
+import React, { useState } from 'react';
+  
+class forgotPassword extends React.Component {
+    constructor() {
+    super();
+    this.state = {
+      input: {},
+      errors: {}
+    };
+     
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+     
+  handleChange(event) {
+    let input = this.state.input;
+    input[event.target.name] = event.target.value;
+  
+    this.setState({
+      input
+    });
+  }
+     
+  handleSubmit(event) {
+    event.preventDefault();
+  
+    if(this.validate()){
+        console.log(this.state);
+
+        let input = {};
+        input["email"] = "";
+        this.setState({input:input});
+  
+        alert('Email confirmed. Form "submitted"');
+    }
+  }
+  
+  validate(){
+      let input = this.state.input;
+      let errors = {};
+      let isValid = true;
+
+      if (!input["email"]) {
+        isValid = false;
+        errors["email"] = "Please enter your email Address.";
+      }
+
+      if (typeof input["email"] !== "undefined") {
```

Comment:
This can just change to "else" (going with the if on line 45) since the condition on 45 will be true if input["email"] is undefined